### ISSUE TITLE
What is `exp_avg` doing as the key of dict `stored_state`?

### ISSUE THREAD

**Issue by shuyueW1991** [Jumpat/SegAnyGAussians#88]
They appeared in method functions of `replace_tensor_to_optimizer`, `_prune_ooptimizer`, etc. What do they do? Because I didn't find exploit calling for them in other scripts.

**Comment by shuyueW1991** [comment]
The key 'exp_avg' in the PyTorch optimizer state dictionary refers to the exponential moving average of gradients. This value is used by optimizers like Adam (Adaptive Moment Estimation) and RMSprop (Root Mean Square Prop) to address issues with noisy gradients during training.
